### DISCUSSION TITLE
Maintenance over the next couple of months

### DISCUSSION THREAD

**Discussion by stefan-k** [argmin-rs/argmin#360] [Announcements]
TLDR: I may be slow to respond during the next months, but the project is still maintained.

Long version:
Starting with June, I will be traveling for a yet unknown number of months. While I will have internet access during that time, other activities than programming will likely have priority. As a result my response time will definitely increase and the amount of time I will be able to spend on argmin may range from none to a lot. If I do find time to work on argmin, I may focus on fun things rather than chores. I am making this announcement to inform you that even if the project seems dead, it is still very much maintained, and I will continue working on it in the future. 

I have been preparing for this trip for months now and this has already limited the time I was able to spend on argmin. Over the last couple of weeks there has been an increased interest in argmin and it makes me a bit sad that I wasn't able to use this momentum. A couple of interesting PRs were opened and important discussions where started but due to the preparations, I wasn't able to give them the attention they deserve. 

I have many ideas on how to proceed with argmin, and I'm dedicated to making them reality. Some of these ideas I want to work on myself, but others may be good starting points for people interested in contributing. For the latter ones I plan to open issues soon-ish. 

I still hope to be able to release a new (minor) version in the beginning of June.

I want to use this opportunity to thank every one who contributed, opened issues, reported bugs and participated in discussions. I really appreciate and enjoy the interaction.
In particular I'd like to thank my sponsors @patefacio and @EdorianDark for their support. It is highly motivating to continue working on such a project when people deem it important enough to actually spend money on it. 

BTW: If anyone in Norway, Sweden or Denmark wants to meet up, feel free to  get in touch :)

**Comment by humphreylee** [comment]
Have a good and safe break.

**Comment by kewei** [comment]
Hi Stefan, this is Kewei, I live in Stockholm, Sweden. I don't know whether you have already come to Sweden. But if you have not, we can have a coffee or beer in Stockholm when you come here, I could also be your guider in Stockholm if it is weekend.

**Reply by stefan-k** [reply]
Hi Kewei,  thanks for the nice offer! I sent you an email :)

**Comment by stefan-k** [comment]
I'm back! :)
